### PR TITLE
2️⃣ refactor(services: webhooks): migrate from fastify to @whatwg-node/server

### PR DESCRIPTION
**Pull request by YassinEldeeb**
related to #657 but doesn't close it entirely

@kamilkisiela How do I test this service(manually and programmatically) to make sure it's working correctly? I haven't seen a contribution guide to follow. All the tests are passing but that doesn't mean it's ready to be merged because webhooks package isn't tested

### CODE REVIEW

File: packages/services/webhooks/src/index.ts [hunk 1]
```diff
@@ -55,56 +54,58 @@ async function main() {
...
+      });
     });
 
-    server.route({
-      method: ['GET', 'HEAD'],
-      url: '/_health',
-      handler(req, res) {
-        void res.status(200).send();
-      },
+    app.get('/_health', () => {
```
Comment (YassinEldeeb) [inline packages/services/webhooks/src/index.ts:82]: this needs to support both `GET` and `HEAD`

**Comment by YassinEldeeb** [comment]
@ardatan just applied your suggestions from slack

**Comment by YassinEldeeb** [comment]
:two:

**Comment by dotansimha** [comment]
Closing. This PR is outdated and we'll reconsider it later.

**Pull request closed without merging**